doc/source/commands/communication.rst

```diff
@@ -0,0 +1,129 @@
+.. _communication:
+
+Communication
+=============
+
+kOS allows you to write scripts that communicate with scripts running on other processors within the same vessel
+(inter-processor communication) or on other vessels (inter-vessel communication).
+
+Messages
+--------
+
+The basic unit of data that is sent between two entities (CPUs or vessels) is called a :struct:`Message`.
+Messages can contain any primitive (scalars, strings, booleans) data types as well as any
+:ref:`serializable <serialization>` types. This allows for a lot of flexibility. It is up to you as a developer
```

Comment:
Maybe we need a term to distinguish "person who works on the mod itself" from "person who uses the mod to make scripts".  Here you mean "script author" when you say "developer" but in other places the word "developer" means working on the C# under the hood.